src/Chat.cs

```diff
@@ -4,48 +4,49 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using OllamaSharp.Models.Chat;
 
 namespace OllamaSharp
 {
-	public class Chat
-	{
-		private List<Message> _messages = new();
+    public class Chat
```

Comment:
Please don't change the formatting to spaces

Reply:
Fixed-